hr_employee_medical_examination/views/hr_employee_medical_examination_views.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="hr_employee_medical_examination_form_view">
+        <field name="name">hr.employee.medical.examination.form (in
+            hr_employee_medical_examination)
+        </field>
+        <field name="model">hr.employee.medical.examination</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <field name="state" widget="statusbar"/>
+                </header>
+                <sheet>
+                    <div class="oe_title">
+                        <label for="name" class="oe_edit_only"/>
+                        <h1>
+                            <field name="name" placeholder="Employee's Name"/>
+                        </h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="employee_id"/>
+                            <field name="date"/>
+                        </group>
+                        <group>
+                            <field name="result"/>
+                        </group>
+                    </group>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" widget="mail_followers"/>
+                    <field name="activity_ids" widget="mail_activity"/>
+                    <field name="message_ids" widget="mail_thread"/>
+                </div>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="hr_employee_medical_examination_search_view">
+        <field name="name">hr.employee.medical.examination.search (in
+            hr_employee_medical_examination)
+        </field>
+        <field name="model">hr.employee.medical.examination</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+                <field name="employee_id"/>
+                <group>
```

Comment:
I was asking by pending filter here:
```suggestion
                <group>
                    <filter string="Pending" name="pending" domain="[('state','=','pending')]"/>
```
It's up to your criteria, non blocking

Reply:
Oh, I missunderstood, just added the pending filter.